src/convertToObject.js

```diff
@@ -1,18 +1,26 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const KEY_INDEX = 0;
+  const VALUE_INDEX = 1;
+  const SPLIT_BY_SYMBOL = `\n`;
+  const SEPARATE_BY_SYMBOL = ':';
+  const MINIMAL_LENGTH_ALLOW = 3;
+
+  sourceString.split(SPLIT_BY_SYMBOL)
+    .filter((currentElement) =>
+      currentElement.length > MINIMAL_LENGTH_ALLOW)
```

Comment:
would be better to filter after splitting by `: `because we possibly can get `style:  ;` with an empty value or vice versa